.github/workflows/android_builds.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         uses: dsaltares/fetch-gh-release-asset@1.1.2
         with:
           repo: darksylinc/godot-swappy
```

Comment:
As suggested in https://github.com/godotengine/godot-build-scripts/pull/106#issuecomment-2695796986, if we move this to the @godotengine org, this should be adjusted here, and here:
```
platform/android/detect.py
190:            "Swappy Frame Pacing not detected! It is strongly recommended you download it from https://github.com/darksylinc/godot-swappy/releases and extract it so that the following files can be found:\n"
```

Reply:
Ok, changed reference from darksylinc/ to godotengine/ here as well.

Hopefully I didn't miss any?

Reply:
Seems good, I grepped for `darksylinc` and the only other references are your credits as contributor, author of betsy, and your V-Sync Simulator tool linked in the docs.